bgp/__init__.py

```diff
@@ -56,7 +56,10 @@
 def _memoize_xml(self):
     if not hasattr(self, '_xml'):
         _memoize_xml_tic = time.perf_counter()
-        self._xml = self.download(formats=['Djvu XML'], return_responses=True)[0].text
+        try:
+            self._xml = self.download(formats=['Djvu XML'], return_responses=True)[0].text
+        except requests.exceptions.Timeout as e:
+            logging.error('Timeout for xml for item - ' + sq.book.identifier + ' | ' + e)
```

Comment:
What is `sq` here?

Reply:
I think this is supposed to be the sequence (https://github.com/Open-Book-Genome-Project/sequencer/pull/58/files#diff-a0ba19faf07afcd67bbf3609db4c8b0f0d5f3ca73ca23a9e2c1e82d1fe300cd9R159), but I don't think it's in scope

Reply:
```suggestion
            logging.error('Timeout for xml for item - ' + self.identifier + ' | ' + e)
```